### ISSUE TITLE
[Suggestion] Show only the channels of currently selected guild

### ISSUE THREAD

**Issue by spode-sleep** [uowuo/abaddon#283]
While sometimes it is useful to see the channels from multiple servers simultaneously, it looks messy in other cases. It would make navigation easier if there was an option to only see the channels from the current guild.

**Comment by ouwou** [comment]
does
```ini
[gui]
classic_channels=true
```
do what you want?

**Comment by spode-sleep** [comment]
It is close enough, but I would rather see text links in the left bar than just icons.

**Comment by ouwou** [comment]
so ideally for you would clicking a server in the normal view auto-collapse whatever other server you had open at the time?

**Comment by spode-sleep** [comment]
Yes, a non-classic menu with an option to keep only currently selected server expanded would be wonderful!
